presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -560,7 +570,13 @@ See below for an example.
         {
           "schema": "default",
           "table": ".*",
-          "privileges": ["SELECT"]
+          "privileges": ["SELECT"],
+          "columns" : [
```

Comment:
Can this be used for `UPDATE` or `INSERT`? What would be the semantic of that?

Reply:
If we add that, then we would alter the allow field to a set of permissions or add a new field.